kong/router.lua

```diff
@@ -425,8 +425,9 @@ local function marshall_route(r)
       match_weight = match_weight + 1
       for i = 1, count do
         local path = paths[i]
+        local is_regex = path:sub(1, 1) == "~"
```

Comment:
`sub` is already localized so:

```suggestion
        local is_regex = sub(path, 1, 1) == "~"
```